src/components/slide-toggle/slide-toggle.ts

```diff
@@ -74,6 +76,14 @@ export class MdSlideToggle implements ControlValueAccessor {
               private _renderer: Renderer) {
   }
 
+  /** @internal */

```

Comment:
We figured out in alpha.5 that we can't actually mark life-cycle events as `@internal`, since they then no longer satisfy the corresponding interface when compiled downstream. So I've been doing `TODO: internal` for when TS adds a first class `internal` access modifier.